### PR TITLE
add clang-tidy

### PR DESCRIPTION
**Pull request by KeisukeShima**

### CODE REVIEW

File: .github/workflows/test-action.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
         path: ${{ env.ARTIFACTS_DIR }}
 
     - id: metrics-reporter
-      uses: tier4/ros-metrics-reporter@main
+      uses: tier4/ros-metrics-reporter@feature/clang-tidy
```
Comment (KeisukeShima) [inline .github/workflows/test-action.yml:59]: TODO: マージ前に戻す

**Comment by KeisukeShima** [comment]
@kmiya I expanded the Artifact obtained from GitHub Actions [(here)](https://github.com/tier4/ros-metrics-reporter/actions/runs/978543879) and confirmed that I can see the result of clang-tidy on the local server.


127 0 0 1_8887_tidy_index html

File: .github/workflows/test-action.yml (hunk 1 above)
Comment (KeisukeShima) [reply .github/workflows/test-action.yml]: DONE

**Comment by kmiya** [comment]
Pretty caught the format issue. Sorry, this is my fault. Could you fix the issue?
image

**Comment by KeisukeShima** [comment]
@kmiya Thanks! fixed in ae6c694

**Review by kmiya — APPROVED** [review]
LGTM :+1:

**Pull request merged**